Client/mods/deathmatch/logic/luadefs/CLuaEngineDefs.cpp

```diff
@@ -503,6 +537,155 @@ int CLuaEngineDefs::EngineImportTXD(lua_State* luaVM)
     return 1;
 }
 
+CClientIMG* CLuaEngineDefs::EngineLoadIMG(lua_State* const luaVM, std::string strFilePath)
+{
+    CLuaMain* pLuaMain = m_pLuaManager->GetVirtualMachine(luaVM);
+    if (!pLuaMain)
+        return false;
```

Comment:
No need to check this.
Also IIRC you have a `getVirtualMachineResource` function in `CLuaManager`.